### ISSUE TITLE
verdi daemon not starting at system startup

### ISSUE THREAD

**Issue by ltalirz** [marvel-nccr/quantum-mobile#62]
in 18.04 the verdi daemon is supposed to start at system startup (but it doesn't )

**Comment by ltalirz** [comment]
also got an error during provisioning 
```
TASK [aiida : Start AiiDA Daemon system service] *******************************************************************
fatal: [default]: FAILED! => {"changed": false, "msg": "Unable to start service aiida-daemon: Job for aiida-daemon.service failed because a configured resource limit was exceeded. See \"systemctl status aiida-daemon.service\" and \"journalctl -xe\" for details.\n"}
```

**Comment by ltalirz** [comment]
Fixed via 1101fb4
